public/static/docs/command-reference/remote/add.md

```diff
@@ -217,28 +217,22 @@ $ dvc remote modify myremote gdrive_client_id my_gdrive_client_id
 $ dvc remote modify myremote gdrive_client_secret gdrive_client_secret
 ```
 
-On first usage of remote you will be prompted to visit access token generation
-link in browser. It will ask you to log into Google account associated with
-Google Drive, which you want to use as DVC's remote. Login process will guide
-you through granting Google Drive access permissions to the used Google Project.
+On first usage of the remote you will be prompted to visit an access token
+generation URL via browser. It will ask you to log into the Google account
+associated with the Google Drive you want to use as remote. The login process
+will guide you through the granting of the required access permissions.
 
-On successful access token generation, token data will be cached in git ignored
-directory with path `.dvc/tmp/gdrive-user-credentials.json`. Do not share token
-data with anyone else to prevent unauthorized access to your Google Drive.
+On successful access token generation, the token data will be cached in a
+Git-ignored directory (located in `.dvc/tmp/gdrive-user-credentials.json`).
 
-#### Shared drives support
```

Comment:
I think it makes to have some kind of title, even if it's a regular bold paragraph

Reply:
Reinstated as well.